cfy_manager/components/components_constants.py

```diff
@@ -52,8 +52,9 @@
 MANAGER_STATUS_REPORTER_USERNAME = 'manager_status_reporter'
 PASSWORD = 'reporter_password'
 TOKEN = 'reporter_token'
-STATUS_REPORTER_ROLE = 'status_reporter'
-
+MANAGER_STATUS_REPORTER_ROLE = 'manager_status_reporter'
+QUEUE_STATUS_REPORTER_ROLE = 'broker_status_reporter'
+DB_STATUS_REPORTER_ROLE = 'db_status_reporter'
```

Comment:
I think the names should be consistent with the [usernames](https://github.com/cloudify-cosmo/cloudify-manager-install/blob/master/cfy_manager/components/components_constants.py#L48). @tyacbovi what do you think?

Reply:
Need to change the usernames to comply with these names